diff --git a/util/log.js b/util/log.js
index 963899e..54a02d8 100644
--- a/util/log.js
+++ b/util/log.js
@@ -1,4 +1,4 @@
-'use strict';
+"use strict";
 
 const winston = require("winston");
 const morgan = require("morgan");
@@ -11,7 +11,7 @@ const logger = new(winston.Logger)({
     transports: [
         new(winston.transports.Console)({
             timestamp: moment(),
-            level: 'silly',
+            level: "silly",
             colorize: true
         })
     ]
@@ -29,11 +29,33 @@ Log.show = (level, message) => {
     }
 };
 
-Log.e = message => Log.show("error", message);
-Log.w = message => Log.show("warn", message);
-Log.i = message => Log.show("info", message);
-Log.v = message => Log.show("verbose", message);
-Log.d = message => Log.show("debug", message);
-Log.s = message => Log.show("silly", message);
+// Error Message
+Log.e = message => {
+    Log.show("error", message);
+}
+// Warning Message
+Log.w = message => {
+    Log.show("warn", message);
+}
+
+// Information Message
+Log.i = message => {
+    Log.show("info", message);
+}
+
+// Verbose Message
+Log.v = message => {
+    Log.show("verbose", message);
+}
+
+// Debug Message
+Log.d = message => {
+    Log.show("debug", message);
+}
+
+// Silly Message
+Log.s = message => {
+    Log.show("silly", message);
+}
 
 module.exports = Log;
\ No newline at end of file
